packages/fast-components-styles-msft/src/design-system/design-system.schema.ts

```diff
@@ -196,16 +196,6 @@ export default {
             type: "number",
             default: 2,
         },
-        neutralForegroundDarkIndex: {
```

Comment:
Assuming we have to remove this based on the tooling - we can't hold off on removing this when we pull the deprecated values because the tooling is wired in to create the form from this, correct?

Reply:
Yeah, it would break the tooling, but I think we’ll be fine without it. It seems safer since it’s not referenced anywhere anymore.

Reply:
👍